torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1658,7 +1673,7 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
           // CALL nodes have already advanced the pc, so
           // undo that to report the call node
           ? frames[i + 1].return_pc - 1
-          : this->pc;
+          : this->saved_pc_;
```

Comment:
`formatStackTrace` is being called before saved_pc_ is restored to the current pc value in the exception handling code. So I think as currently written this is going to misreport the location of all exceptions. Generally I am worried about how fragile this approach will be over time. For context, the interpreter used to have an `ActivateFrame` object lived only in the interpreter loop function and held the PC as well as some pre-computed values. We removed it because it was more difficult to maintain than any performance it provided.